### ISSUE TITLE
Force the use of HTTPS in M1/L1/E2/T2/S2 since HTTP will not work.

### ISSUE THREAD

**Issue by tinodo** [MicrosoftLearning/AZ-204-DevelopingSolutionsforMicrosoftAzure#570]
# Module: 01
##### Step: 02

Description of issue

The instructions, next to "**Value** text box" read: _Make sure you include the protocol, such as **https://**, in the URL that you copy into the Value text box for this application setting_

This is **INCORRECT**. You **HAVE** to use **https://**. http:// will not work.
Using HTTP:// as the protocol, you WILL see existing images on the storage account, but you will not be able to upload new ones.

Repro steps:

1. On the mentioned step, http://imgapixxx.azurewebsites.net for the ApiUrl value.
2. Finish the lab and try to upload a picture.

Of course, you can also change the code of the WebApi to make sure that http is supported as well.

**Comment by msftnutta** [comment]
@tinodo  I've tested it and even our lab instruction suggests to use **HTTPS://** not **HTTP://**

and both App Service - Web App and Storage account will have **HTTPS://** enabled by default? I do not see anywhere in this lab that we would use **http://**

Could you point out again regarding the problem? Which environment you see this **http://** problem? Azure Pass Subscription, CloudSlice, which language? as there are many variants of our labs at the moment.

**Comment by tinodo** [comment]
You are right, it SUGGESTS to use https://, and https is enabled by default, however, it doesn't clearly state you HAVE to use https:

_Make sure you include the protocol, **such as** https://, in the URL that you copy into the Value text box for this application setting_

This sentence implies you could use 'another' protocol. Just rephrasing the sentence to:

_Make sure you include the protocol, https://, in the URL that you copy into the Value text box for this application setting_

would be enough!

**Comment by msftnutta** [comment]
Thanks @tinodo i've created a pull request #571  @JeffKoMS  it's a very minor issue and I agree with Tino, Please review it, thanks.
